client/index.html

```diff
@@ -122,75 +122,81 @@ <h1 class="title">Sign in to The Lounge</h1>
 						<div class="row">
 							<div class="col-sm-12">
 								<h1 class="title">
-									<%= public ? "The Lounge - " : "" %>
+									{{#if public}}The Lounge - {{/if}}
 									Connect
-									<%= !displayNetwork && lockNetwork ? "to " + defaults.name : "" %>
+									{{#unless displayNetwork}}
+										{{#if lockNetwork}}
+										to {{defaults.name}}
+										{{/if}}
+									{{/unless}}
 								</h1>
 							</div>
-							<div <%= typeof(displayNetwork) !== "undefined" && !displayNetwork ? 'style="display: none;"' : ''%>>
+							{{#unless displayNetwork}}
```

Comment:
We've changed our config long ago to be bootstrapped from example config, so missing keys in users config are set to default values.